gdal/swig/python/scripts/gcps2vec.py

```diff
@@ -1,9 +1,9 @@
 import sys
 # import osgeo.utils.gcps2vec as a convenience to use as a script
-from osgeo.utils.gcps2vec import *
+from osgeo.utils.gcps2vec import *  # noqa
```

Comment:
why do we need this wildcard import ? Isn't the below import of main sufficient ?

Reply:
> I see some tests added gdal_calc, for instance, to the path then imported it as a module. It's ugly, and with this PR it's unnecessary. I've changed this particular test to import the module from osgeo.utils. But I guess that if we want to maintain complete backwards compatibility this import star is needed.

OK